ods_ci/tests/Resources/Page/Operators/OpenShiftPipelines.resource

```diff
@@ -10,17 +11,44 @@ Library             ../../libs/DataSciencePipelinesAPI.py
 
 
 *** Variables ***
-${REDHAT_OPENSHIFT_PIPELINES_YAML}=            ods_ci/tests/Resources/Files/redhat-openshift-pipelines.yaml
-&{OPENSHIFT_PIPELINES_VERSION}=                4.10=pipelines-1.8   4.11=pipelines-1.9  4.12=pipelines-1.10   4.13=pipelines-1.10  #robocop:disable
+${REDHAT_OPENSHIFT_PIPELINES_YAML}      ods_ci/tests/Resources/Files/redhat-openshift-pipelines.yaml
 
 
 *** Keywords ***
+# robocop: disable=too-many-calls-in-keyword,line-too-long
 Install Red Hat OpenShift Pipelines
-    [Documentation]    Install Red Hat OpenShift Pipelines
-    ${oc_vesrion}    Get OpenShift Version
-    ${pipeline}   Get From Dictionary     ${OPENSHIFT_PIPELINES_VERSION}    ${oc_vesrion}
-    ${return_code}    ${output}    Run And Return Rc And Output    sed -i "s,channel: .*,channel: ${pipeline},g" ${EXECDIR}/${REDHAT_OPENSHIFT_PIPELINES_YAML}    #robocop:disable
-    Should Be Equal As Integers	 ${return_code}	 0
+    [Documentation]    Install Red Hat OpenShift Pipelines operator depending on
+    ...     OpenShift version, platform and RHODS flavor (Managed/Self-Managed):
+    ...     - OpenShift 4.10:                  OpenShift Pipelines 1.8
+    ...     - OpenShift 4.11:                  OpenShift Pipelines 1.9
+    ...     - OpenShift >= 4.12 (AWS and GCP): OpenShift Pipelines 1.10 / 1.11 (depending on platform and flavor)
+    ...     - OpenShift >= 4.12 (other):       OpenShift Pipelines 1.11
+    ${oc_version}=    Get OpenShift Version
+    ${is_self_managed}=    Is RHODS Self-Managed
+    ${platform}=    Fetch Cluster Platform Type
+    IF    ${oc_version} == "4.10"
+        ${openshift_pipelines_version}=    Set Variable    pipelines-1.8"
+    ELSE IF    ${oc_version} == "4.11"
+        ${openshift_pipelines_version}=    Set Variable    pipelines-1.9
+    ELSE
+        IF    ${is_self_managed} == ${TRUE} and "${platform}" == "AWS"
+            ${openshift_pipelines_version}=    Set Variable    pipelines-1.10
+        ELSE IF    ${is_self_managed} == ${FALSE} and "${platform}" == "AWS"
+            ${openshift_pipelines_version}=    Set Variable    pipelines-1.11
```

Comment:
why SM and CS requires 2 different versions of the pipelines operator?

Reply:
They don't. This logic is to ensure that we test OpenShift Pipelines 1.10 and 4.11 in AWS and GCP without having to run all possible combinations

Reply:
I've updated the [Documentation] section to explain this